chromium/package.json

```diff
@@ -12,7 +12,7 @@
     "fetch-mock": "^6.3.0",
     "mocha": "^4.1.0",
     "nan": "^2.10.0",
-    "node-webcrypto-ossl": "^1.0.36",
+    "node-webcrypto-ossl": "^1.0.37",
```

Comment:
I'm aware that this line is not absolutely necessary due to `^`, which would use v1.0.38 currently. Specifying this particular version is to relate https://github.com/EFForg/https-everywhere/issues/15253.